### DISCUSSION TITLE
Recover old SpecFlow repositories

### DISCUSSION THREAD

**Discussion by icnocop** [reqnroll/Reqnroll#359] [General]
Hi.

We're still in the process of migrating to Reqnroll.

We've found the old SpecFlow repositories have been deleted.

All the google search results return 404 Not Found
https://github.com/techtalk/SpecFlow
https://github.com/SpecFlowOSS/SpecFlow
https://github.com/SpecFlowOSS/SpecFlow.VS

Even the community forums are gone
https://support.specflow.org/hc/en-us/community/topics 

I'd like to please request recovering the repository and either archiving it, or transfer it to another account.

Thank you.

**Comment by ajeckmans** [comment]
Just so you know. No one here is part of Tricentis or has anything to do with the current affairs of Specflow. It is beyond our power to restore it.

**Reply by icnocop** [reply]
Do you know how I can get in touch with someone who can? Thank you.

**Reply by gasparnagy** [reply]
Unfortunately not. I had contacts, but they are not replying to me anymore for some time.

**Comment by Romfos** [comment]
Why do we need it? Last commit was in 2022. Without support It doesn't work for modern .NET like .NET 8 and .NET 9.
Better to contribute in reqnroll

**Reply by icnocop** [reply]
We're still in the process of migrating.

Google search results point to issues in those repositories and comments made by community members, for example, may help resolve similar issues in reqnroll.

**Reply by icnocop** [reply]
The latest SpecFlow.VS extension release had support for .NET 8.

But since the repository was deleted, the download is no longer available.

It still worked for projects targeting .NET 4.

**Reply by gasparnagy** [reply]
The `reqnroll_fork` tags both on the Reqnroll and the Reqnroll.VisualStudio codebase point to the last changes of the SpecFlow codebase, if you technically need to access the code, but unfortunately the issues and all the comments there are gone. ☹️ Incredible.

The Reqnroll Visual Studio extension should be able to load and work with SpecFlow projects still, so you can try that.

**Reply by gasparnagy** [reply]
I asked it on LinkedIn, maybe someone still have the VSIX file.

**Reply by WilcoB-Centric** [reply]
I might have it Gaspar, on my personal machine. I'll look at it tomorrow

**Comment by Tiberriver256** [comment]
You can find the forks which aren't usually returned in GitHub search.

```
Specflow fork:only
```

**Comment by gasparnagy** [comment]
See related posts:
* LinkedIn: https://www.linkedin.com/posts/gasparnagy_github-reqnrollreqnroll-open-source-cucumber-style-activity-7274869345271312384-Avos?utm_source=share&utm_medium=member_desktop
* Reddit: https://www.reddit.com/r/dotnet/comments/1hgyxrp/specflow_is_no_longer_open_source/

**Comment by icnocop** [comment]
It seems the repositories are back! 🙏

But the issues and releases are not. 😭

**Comment by tmasa** [comment]
You can find the VSIX of the latest version of the Visual Studio extension (the one that supports .NET8) attached below: [SpecFlow.VisualStudio.Package.2022.1.93.zip](https://github.com/user-attachments/files/18206231/SpecFlow.VisualStudio.Package.2022.1.93.zip)

**Comment by gasparnagy** [comment]
FYI: Although the GitHub repositories have been recovered (only the code, not the issues and releases), they will be removed again after 1st of January according to the official announcement of Tricentis: https://support-hub.tricentis.com/open?number=NEW0001432&id=post
